### ISSUE TITLE
Nano: trainer with use_ipex=true is slower than original Nano trainer

### ISSUE THREAD

**Issue by y199387** [intel-analytics/BigDL#5051]
## Environment:
Pytorch 1.11.0
torchvision 0.12.0
intel extension for pytorch 1.11.0
## Problem
I test the script that training resnet50, the result is:
- origin trainer: 1373
- trainer with use_ipex=True: 1458
```python
import torch
import torchvision

LR = 0.001
DATA = "/home/data/"

transform = torchvision.transforms.Compose([
    torchvision.transforms.Resize((224, 224)),
    torchvision.transforms.ToTensor(),
    torchvision.transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))
])
train_dataset = torchvision.datasets.CIFAR10(
        root=DATA,
        train=True,
        transform=transform,
        download=True,
)
train_loader = torch.utils.data.DataLoader(
        dataset=train_dataset,
        batch_size=128
)

from bigdl.nano.pytorch import Trainer
model = torchvision.models.resnet50()
model = model.to(memory_format=torch.channels_last)
criterion = torch.nn.CrossEntropyLoss()
optimizer = torch.optim.SGD(model.parameters(), lr=LR, momentum=0.9)
model = Trainer.compile(model, criterion, optimizer)
trainer = Trainer(max_epochs=1)

from time import time
start = time()
trainer.fit(model, train_loader)
basic = time() - start
print(basic)

ipex_model = torchvision.models.resnet50()
ipex_model = ipex_model.to(memory_format=torch.channels_last)
ipex_criterion = torch.nn.CrossEntropyLoss()
ipex_optimizer = torch.optim.SGD(ipex_model.parameters(), lr=LR, momentum=0.9)
ipex_model = Trainer.compile(ipex_model, ipex_criterion, ipex_optimizer)
ipex_trainer = Trainer(max_epochs=1, use_ipex=True)

start = time()
ipex_trainer.fit(ipex_model, train_loader)
ipex_fit_time = time() - start
print(ipex_fit_time)
```

As a comparison, I also tested the example(same model, loss and optimizer) given by ipex from [here](https://intel.github.io/intel-extension-for-pytorch/1.11.200/tutorials/examples.html#complete-float32)

- model without ipex.optimize: 1388
- model with ipex.optimize: 870

```python
import torch
import torchvision
import intel_extension_for_pytorch as ipex

LR = 0.001
DATA = "/home/data/"

transform = torchvision.transforms.Compose([
    torchvision.transforms.Resize((224, 224)),
    torchvision.transforms.ToTensor(),
    torchvision.transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))
])
train_dataset = torchvision.datasets.CIFAR10(
        root=DATA,
        train=True,
        transform=transform,
        download=True,
)
train_loader = torch.utils.data.DataLoader(
        dataset=train_dataset,
        batch_size=128
)

model = torchvision.models.resnet50()
model = model.to(memory_format=torch.channels_last)
criterion = torch.nn.CrossEntropyLoss()
optimizer = torch.optim.SGD(model.parameters(), lr = LR, momentum=0.9)
model.train()
optimized_model, optimized_optimizer = ipex.optimize(model, optimizer=optimizer)

from time import time
start = time()
for batch_idx, (data, target) in enumerate(train_loader):
    # Setting memory_format to torch.channels_last could improve performance with 4D input data. This is optional.
    data = data.to(memory_format=torch.channels_last)
    optimizer.zero_grad()
    output = model(data)
    loss = criterion(output, target)
    loss.backward()
    optimizer.step()
    print(batch_idx)
normal = time() - start
print(normal)

start = time()
for batch_idx, (data, target) in enumerate(train_loader):
    # Setting memory_format to torch.channels_last could improve performance with 4D input data. This is optional.
    data = data.to(memory_format=torch.channels_last)
    optimized_optimizer.zero_grad()
    output = optimized_model(data)
    loss = criterion(output, target)
    loss.backward()
    optimized_optimizer.step()
    print(batch_idx)
fit_time = time() - start
print(fit_time)
```

**Comment by y199387** [comment]
close as solved
